source/common/tracing/opentracing_driver_impl.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <memory>
+
+#include "envoy/tracing/http_tracer.h"
+
+#include "common/common/logger.h"
+
+#include "opentracing/tracer.h"
+
+namespace Envoy {
+namespace Tracing {
+
+class OpenTracingSpan : public Span, Logger::Loggable<Logger::Id::tracing> {
```

Comment:
Can we have some links to docs here or just a set of comments somewhere on how a driver is actually created? It's a little confusing to me re: what the various propagation modes are, what the driver implements, how the header lookup code works (in terms of what the OT code expects), etc.

Reply:
Will do.

Reply:
Added documentation.